src/Tools/IdeBenchmarks/SQLitePersistentStorageBenchmark.cs

```diff
@@ -66,12 +66,12 @@ public void GlobalSetup()
                 .WithChangedOption(StorageOptions.Database, StorageDatabase.SQLite)));
 
             var connectionPoolService = _workspace.ExportProvider.GetExportedValue<SQLiteConnectionPoolService>();
-            var asyncListener = _workspace.ExportProvider.GetExportedValue<IAsynchronousOperationListenerProvider>().GetListener(FeatureAttribute.PersistentStorage);
+            var asyncListener = _workspace.ExportProvider.GetExportedValue<IAsynchronousOperationListenerProvider>();
 
-            _storageService = new SQLitePersistentStorageService(connectionPoolService, new StorageConfiguration(), asyncListener);
+            _storageService = new SQLitePersistentStorageService(connectionPoolService, asyncListener);
 
             var solution = _workspace.CurrentSolution;
-            _storage = _storageService.GetStorageWorkerAsync(SolutionKey.ToSolutionKey(solution), CancellationToken.None).AsTask().GetAwaiter().GetResult();
+            _storage = _storageService.GetStorageWorkerAsync(new StorageConfiguration(), SolutionKey.ToSolutionKey(solution), CancellationToken.None).AsTask().GetAwaiter().GetResult();
```

Comment:
instead of 'configuration' being part of the Service itself, it is now part of Storage.  This allows the Service to be a singleton, but individual Storage instances may be affected by the config passed in.